### PR TITLE
Separated development from custom content and add gamereplays.org

### PR DESCRIPTION
**Pull request by Mailaender**
Using https://commons.wikimedia.org/wiki/File:Gnome-emblem-shared.svg as icon.

### CODE REVIEW

File: content/community.html [hunk 1]
```diff
@@ -17,6 +17,14 @@
...
+<p>We are a small team, so anything you can do to help is greatly appreciated.</p>
+<p>See the <a href="http://wiki.openra.net/#development-topics" alt="development topics">development topics</a> on our wiki for more information.</p>
+</div>
+
+<hr />
+<h3>Getting Content</h3>
+<div class="communityblock">
+<img src="/images/shared.svg" alt="Sharing" title="Sharing" />
+<p>You can share custom content such as maps on the <a href="http://resource.openra.net/">OpenRA Resource Center</a> site.<br />
+Replays, strategies and videos can be published at <a href="http://www.gamereplays.org/openra/">GameReplays.org</a> OpenRA section.</p>
```
Comment (obrakmann) [inline content/community.html]: either "at *the* GameReplays.org OpenRA section" or "at GameReplays.org's OpenRA section"

**Comment by obrakmann** [comment]
:+1: aside from that little grammar nit

File: content/images/shared.svg [hunk 2]
```diff
@@ -0,0 +1,552 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (pchote) [inline content/images/shared.svg]: The open hand has been effectively deprecated as a sharing symbol.  https://bold.pixelapse.com/minming/share-the-icon-no-one-agrees-on presents some better alternatives.

I agree with his conclusion of the "outgoing tray" icon being the best general-purpose glyph for this.

File: content/community.html [hunk 3]
```diff
@@ -17,6 +17,14 @@
...
 <div class="communityblock">
 <img src="/images/development.svg" alt="Development" title="Development" />
-<p>We are a small team, so anything you can do to help is greatly appreciated.<br /> See the <a href="http://wiki.openra.net/#development-topics" alt="development topics">development topics</a> on our wiki for more information.</p>
-<p>You can share custom content such as maps on the <a href="http://resource.openra.net/">OpenRA Resource Center</a> site.</p>
+<p>We are a small team, so anything you can do to help is greatly appreciated.</p>
+<p>See the <a href="http://wiki.openra.net/#development-topics" alt="development topics">development topics</a> on our wiki for more information.</p>
+</div>
+
+<hr />
+<h3>Getting Content</h3>
```
Comment (pchote) [inline content/community.html]: The headline is "Getting Content", but the body is all about *sharing* content.

**Comment by pchote** [comment]
Don't forget my two comments above.

File: content/community.html (hunk 3 above)
Comment (Mailaender) [inline content/community.html]: I changed sharing to getting in the line below. For most people those sites are about consuming.

**Comment by Mailaender** [comment]
I tried that already. Keeping the Tango icon and separating game development from content creation seems like the best solution already.

File: content/community.html (hunk 3 above)
Comment (pchote) [inline content/community.html]: This now has two gets (which isn't very good english) and a share.

**Comment by pchote** [comment]
The hand icon is a worse fit for getting content than for sharing, IMO.  The first association that comes to my mind is beggers on the street.

**Comment by Mailaender** [comment]
I replaced it with https://commons.wikimedia.org/wiki/File:Gnome-emblem-downloads.svg

**Comment by Mailaender** [comment]
See also https://github.com/ihptru/OpenRA-Content-Website/issues/152.

**Comment by obrakmann** [comment]
Works for me :+1:

**Pull request merged**